### DISCUSSION TITLE
How does one configure Renovate to automerge `.nvmrc` updates (all versions) if it's green?

### DISCUSSION THREAD

**Discussion by rodoabad** [renovatebot/renovate#22960] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate automatically updates `.nvmrc` whenever there's a Node update.

### What you tried so far.

I read this https://docs.renovatebot.com/modules/manager/nvm/ and it says Renovate already does it by default but so far all I'm seeing is Renovate updating Docker files and not `.nvmrc`.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Renovate never automerges by default. You need to configure it with packageRules
